Packs/Twitter/Integrations/Twitterv2/Twitterv2.py

```diff
@@ -0,0 +1,415 @@
+    for dict_value in dict_list:
+        dict_to_append = {}
+        for key, value in dict_value.items():
+            if isinstance(value, dict):
+                dict_to_append.update({f'{key}.{k}': v for k, v in value.items()})
+            else:
+                dict_to_append[key] = value
+        list_dict_response.append(dict_to_append)
+    return list_dict_response
+
+
+def header_transform_tweet_search(header: str) -> str:
+    """ Gets header and transform it to another header according to tweet_search command.
+        Args:
+            header: str -  A header string.
+        Returns:
+            A header string.
+    """
+    if header == 'id':
+        return 'Tweet ID'
+    if header == 'text':
+        return 'Text'
+    if header == 'created_at':
+        return 'Created At'
+    if header == 'author.name':
+        return 'Author Name'
+    if header == 'author.username':
+        return 'Author Username'
+    if header == 'public_metrics.like_count':
+        return 'Likes Count'
+    if header == 'next_token':
+        return 'Next Token'
+    if header == 'media.url':
+        return 'Attachments URL'
+    else:
+        return stringEscapeMD(header, True, True)
+    return ""
+
+
+def header_transform_get_user(header: str) -> str:
+    """ Gets header and transform it to another header according to get_user command.
+        Args:
+            header: str -  A header string.
+        Returns:
+            A header string.
+    """
+    if header == 'name':
+        return 'Name'
+    if header == 'username':
+        return 'User name'
+    if header == 'created_at':
+        return 'Created At'
+    if header == 'description':
+        return 'Description'
+    if header == 'public_metrics.followers_count':
+        return 'Followers Count'
+    if header == 'public_metrics.tweet_count':
+        return 'Tweet Count'
+    if header == 'verified':
+        return 'verified'
+    else:
+        return stringEscapeMD(header, True, True)
+    return ""
+
+
+def twitter_tweet_search_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """ Gets args and client and returns List[CommandResults] of Tweets according to the reqested search.
+        Args:
+            client: client -  A Twitter client.
+            args: Dict - The function arguments.
+        Returns:
+            A List of CommandResult objects with users data according to to the reqested.
+    """
+    headers = ['id', 'text', 'created_at', 'author.name', 'author.username', 'public_metrics.like_count', 'media.url']
+    query = argToList(args.get('query'))
+    if not query:
+        raise ValueError('query not specified')
+    start_time = date_to_iso_format(args.get('start_time', '')) if args.get('start_time', None) else None
+    end_time = date_to_iso_format(args.get('end_time', '')) if args.get('end_time', None) else None
+    limit = arg_to_number(args.get('limit', 50))
+    limit = 100 if limit and limit >= 100 else limit
```

Comment:
Why are we doing this? 
We just need to do a check that the limit is between 10 and 100 no?
If the number is bigger, raise Exception